app/src/main/java/edu/uc/groupproject/covid19tracker/ui/main/ItemAdapter.kt

```diff
@@ -21,20 +23,21 @@ class ItemAdapter(var context: Context, var view_layout: Int, var arrayCaseList:
         return arrayCaseList.size
     }
 
+    @SuppressLint("SetTextI18n", "ViewHolder")
     override fun getView(position: Int, convertVire: View?, parent: ViewGroup?): View {
         val view: View = View.inflate(context, R.layout.country_list_view_layout, null)
 
-        var title: TextView = view.findViewById<TextView>(R.id.list_country)
-        var cases: TextView = view.findViewById<TextView>(R.id.list_cases)
-        var recovered: TextView = view.findViewById<TextView>(R.id.list_recovered)
-        var deaths: TextView = view.findViewById<TextView>(R.id.list_deaths)
+        val title: TextView = view.findViewById(R.id.list_country)
+        val cases: TextView = view.findViewById(R.id.list_cases)
+        val recovered: TextView = view.findViewById(R.id.list_recovered)
+        val deaths: TextView = view.findViewById(R.id.list_deaths)
```

Comment:
Gets rid of some redundancy: https://kotlinlang.org/docs/reference/coding-conventions.html#avoiding-redundant-constructs
Also changes variables to constants

Reply:
findViewById is old school; not needed in Kotlin (except in one case: recycler views).  Use Kotlin synthetic imports, and you can remove this entire block.  Less code == less maintenance.